Source/WebGPU/WGSL/AST/ASTIdentifierExpression.h

```diff
@@ -26,26 +26,27 @@
 #pragma once
 
 #include "ASTExpression.h"
-
-#include <wtf/text/WTFString.h>
+#include "ASTIdentifier.h"
 
 namespace WGSL::AST {
 
 class IdentifierExpression final : public Expression {
```

Comment:
It feels a bit weird how similar this is to `AST::Identifier` now, but maybe it's inevitable?